Source/WTF/wtf/HashTable.h

```diff
@@ -1051,16 +999,16 @@ DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(HashTable);
     inline auto HashTable<Key, Value, Extractor, HashFunctions, Traits, KeyTraits>::reinsert(ValueType&& entry) -> ValueType*
     {
         ASSERT(m_table);
-        ASSERT(!lookupForWriting(Extractor::extract(entry)).second);
-        ASSERT(!isDeletedBucket(*(lookupForWriting(Extractor::extract(entry)).first)));
+        ASSERT(!isDeletedBucket(*(lookupForReinsert(Extractor::extract(entry)))));
+        ASSERT(isEmptyBucket(*(lookupForReinsert(Extractor::extract(entry)))));
```

Comment:
I would have omitted the parentheses after the `*` here. I think they actually make the code slightly harder to read.

Reply:
Fixed.